src/Psy/Command/ParseCommand.php

```diff
@@ -126,7 +126,9 @@ private function parse($code)
     private function getParser()
     {
         if (!isset($this->parser)) {
-            $this->parser = new Parser(new Lexer());
+            $this->parser = (new ParserFactory())->create(
```

Comment:
you can't just update the version constraint to allow 1.x again, but leave the code here that only works on 2.x